fdbserver/workloads/ChangeConfig.actor.cpp

```diff
@@ -32,13 +32,15 @@ struct ChangeConfigWorkload : TestWorkload {
 	double minDelayBeforeChange, maxDelayBeforeChange;
 	std::string configMode; //<\"single\"|\"double\"|\"triple\">
 	std::string networkAddresses; // comma separated list e.g. "127.0.0.1:4000,127.0.0.1:4001"
+	int coordinatorChanges; // number of times to change coordinators. Only applied if `coordinators` is set to `auto`
 
 	ChangeConfigWorkload(WorkloadContext const& wcx) : TestWorkload(wcx) {
 		minDelayBeforeChange = getOption(options, LiteralStringRef("minDelayBeforeChange"), 0);
 		maxDelayBeforeChange = getOption(options, LiteralStringRef("maxDelayBeforeChange"), 0);
 		ASSERT(maxDelayBeforeChange >= minDelayBeforeChange);
 		configMode = getOption(options, LiteralStringRef("configMode"), StringRef()).toString();
 		networkAddresses = getOption(options, LiteralStringRef("coordinators"), StringRef()).toString();
+		coordinatorChanges = getOption(options, LiteralStringRef("coordinatorChanges"), 1);
```

Comment:
I think it may be simpler to set `coordinatorChanges = 0` here if `networkAddresses != "auto";` This would simplify the code of `ChangeConfigClient`.

Reply:
We still want to do one coordinator change if `networkAddresses != "auto"` (at least that's how the current code works), so I think it should be 1 instead of 0. But I see your idea, I added this and simplified the coordinator change code a bit.